src/audio/eq_fir/eq_fir.c

```diff
@@ -75,39 +65,39 @@ struct comp_data {
 #if CONFIG_FORMAT_S16LE
 static inline void set_s16_fir(struct comp_data *cd)
 {
-	cd->eq_fir_func = eq_fir_2x_s16_hifi3;
+	cd->eq_fir_func = eq_fir_2x_s16;
 }
 #endif /* CONFIG_FORMAT_S16LE */
 #if CONFIG_FORMAT_S24LE
 static inline void set_s24_fir(struct comp_data *cd)
 {
-	cd->eq_fir_func = eq_fir_2x_s24_hifi3;
+	cd->eq_fir_func = eq_fir_2x_s24;
 }
 #endif /* CONFIG_FORMAT_S24LE */
 #if CONFIG_FORMAT_S32LE
 static inline void set_s32_fir(struct comp_data *cd)
 {
-	cd->eq_fir_func = eq_fir_2x_s32_hifi3;
+	cd->eq_fir_func = eq_fir_2x_s32;
 }
 #endif /* CONFIG_FORMAT_S32LE */
 
 #elif FIR_HIFIEP
```

Comment:
is this macro still needed if the functions are all the same name now?

Reply:
Yep, I need to clean up this.

Reply:
Done!